.github/workflows/ci.yml

```diff
@@ -112,3 +112,13 @@ jobs:
         uses: github/codeql-action/upload-sarif@codeql-bundle-20211208
         with:
           sarif_file: trivy-results.sarif
+
+      - name: Upload binaries to release
+        if: ${{ github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master' }}
```

Comment:
Shouldn't be the ref a tag and not master or develop?

Reply:
Fixed with `github.tag`